app/src/main/java/org/oppia/android/app/databinding/MarginBindingAdapters.java

```diff
@@ -14,12 +14,7 @@ public final class MarginBindingAdapters {
   public static void setLayoutMarginStart(@NonNull View view, float marginStart) {
     if (view.getLayoutParams() instanceof MarginLayoutParams) {
       MarginLayoutParams params = (MarginLayoutParams) view.getLayoutParams();
-      float marginEnd = params.getMarginEnd();
-      if (isRtlLayout(view)) {
-        setLayoutDirectionalMargins(view, (int) marginEnd, (int) marginStart);
-      } else {
-        setLayoutDirectionalMargins(view, (int) marginStart, (int) marginEnd);
-      }
+      MarginLayoutParamsCompat.setMarginStart(params, (int) marginStart);
```

Comment:
Could you add an explanation in your PR description for why we made the change to `MarginLayoutParamsCompat`